test/cpp/api/modules.cpp

```diff
@@ -1661,6 +1661,15 @@ TEST_F(ModulesTest, PrettyPrintBatchNorm) {
       "torch::nn::BatchNorm(features=4, eps=0.5, momentum=0.1, affine=false, stateful=true)");
 }
 
+TEST_F(ModulesTest, PrettyPrintLayerNorm) {
+  ASSERT_EQ(
+    c10::str(LayerNorm(LayerNormOptions({2, 2}))),
+      "torch::nn::LayerNorm([2, 2], eps=1e-05, elementwise_affine=true)");
+      ASSERT_EQ(
+        c10::str(LayerNorm(LayerNormOptions({2, 2}).elementwise_affine(false).eps(2e-5))),
+          "torch::nn::LayerNorm([2, 2], eps=2e-05, elementwise_affine=false)");
+}
```

Comment:
It would be awesome to add a test for `LayerNorm` module forward as well, similar to other modules in this test file